tag.hbs

```diff
@@ -15,11 +15,14 @@
             <img class="gh-page-image" src="{{feature_image}}" alt="{{name}}" />
         {{/if}}
     </header>
-    {{/tag}}
 
     <div class="grid grid-cols-1 lg:grid-cols-3 gap-x-20 gap-y-10 w-4/5 my-4 mx-auto">
-    {{#foreach posts}}
-        {{> "post-card"}}
-    {{/foreach}}
+    {{#get "posts" limit="all" filter="tags:{{slug}}" include="tags" as |tagged|}}
+        {{#foreach tagged}}
+            {{> "post-card"}}
+        {{/foreach}}
+        {{else}} <p>There are no related articles.</p>
```

Comment:
Also, this should be `There are no articles with this tag.` For `author.hbs`, `There are no articles written by this author.`